.github/workflows/nightly-build.yaml

```diff
@@ -10,7 +10,7 @@ permissions:
   contents: write
 jobs:
   nightly-build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Not to rain on any parades but it looks like this dropped darwin-arm64: https://github.com/jozu-ai/kitops/releases/tag/next

Reply:
good catch! thanks!